lib/config/fileconf.go

```diff
@@ -982,6 +983,10 @@ type Auth struct {
 	// LoadAllCAs tells tsh to load the CAs for all clusters when trying
 	// to ssh into a node, instead of just the CA for the current cluster.
 	LoadAllCAs bool `yaml:"load_all_cas,omitempty"`
+
+	// HostedPlugins configures the hosted plugins runtime.
+	// This is currently Cloud-specific.
+	HostedPlugins HostedPlugins `yaml:"hosted_plugins,omitempty"`
```

Comment:
Since we're going to have a separate `plugin_service` (see OpsGenie RFD), can we make this a part of its configuration right away?

```yaml
plugin_service:
  enabled: yes
  opsgenie:
    api_key: /home/ubuntu/key
  slack:
    client_id:
    client_secret:
```

Reply:
As discussed in Slack, self-hosted versions of Slack plugin can keep using auth tokens so this configuration is specific to Cloud's hosted plugins use-case and can probably stay here for now.